pkgs/by-name/pi/piano-rs/package.nix

```
@@ -0,0 +1,48 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  makeWrapper,
+  pkg-config,
+  alsa-lib,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "piano-rs";
+  version = "0.2.0";
+
+  src = fetchFromGitHub {
+    owner = "ritiek";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-qZeH9xXQPIOJ87mvLahnJB3DuEgLX0EAXPvECgxNlq0=";
+  };
+
+  cargoHash = "sha256-vDqfWXeQVEnMWMjhAG/A0afff7dWMoQejDZjcVlYBMQ=";
+
+  nativeBuildInputs = [
+    makeWrapper
+    pkg-config
+  ];
+
+  buildInputs = [
+    alsa-lib
+  ];
+
+  postInstall = ''
+    mkdir -p "$out"/lib
+    install -Dm644 ${alsa-lib.out}/lib/libasound* "$out"/lib/
+
+    cp -r assets "$out"/
+    wrapProgram "$out"/bin/piano-rs --set ASSETS "$out"/assets
```

I'd suggest moving this in a sub directory. For example:

```suggestion
    mkdir -p "$out"/share/piano-rs
    cp -r assets "$out"/share/piano-rs
    wrapProgram "$out"/bin/piano-rs \
      --set ASSETS "$out"/share/piano-rs/assets
```